packages/datetime/src/dateRangePicker.tsx

```diff
@@ -80,6 +80,12 @@ export interface IDateRangePickerProps extends IDatePickerBaseProps, IProps {
      */
     defaultValue?: DateRange;
 
```

Comment:
please make sure to add to the JSDoc comment for the `dayPickerProps` prop, just like in `datePicker.tsx`:


```
     * In case of supplying your owner `renderDay` function, make sure to apply the appropriate
     * CSS wrapper class to obtain default Blueprint styling.
     * eg.
     * `<div className={Classes.DATEPICKER_DAY_WRAPPER}>{CONTENT_HERE}</div>`
```